handler/task/interface.go

```diff
@@ -0,0 +1,14 @@
+package task
+
+import (
+	Models "awesomeProject/models"
+	"gofr.dev/pkg/gofr"
+)
+
+type TaskService interface {
+	AddTask(ctx *gofr.Context, task string, uid int) error
```

Comment:
We could have passed the task struct to add a task instead of a desc string and an id int. Creating a new task structure again in the service layer seems memory waste.

This is a suggestion in design, not required to change as of now, but is a good practice.